### PR TITLE
Adding web binding cert store tests

### PR DESCRIPTION
**Pull request by arkasha**
This is a test and fix for https://github.com/PowerShell/xWebAdministration/issues/13

### CODE REVIEW

File: Tests/MSFT_xWebBindingInformation.Tests.ps1 [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+Describe "MSFT_xWebBindingInformation" {
+    It 'Should be able to get xWebsite' -test {
+        # just a good idea.  
+        # I thought it might force the classes to register, but it does not.
+        $resources = Get-DscResource -Name xWebsite
+        $resources.count | should be 1
+    }
+
+    It 'Should compile and run without throwing (But does not currently forcing pass)' -test {
```
Comment (arkasha) [inline Tests/MSFT_xWebBindingInformation.Tests.ps1]: There is a current issue open for this: https://github.com/PowerShell/xWebAdministration/issues/17

**Comment by arkasha** [comment]
Please review this pull request. Thanks!

File: appveyor.yml [hunk 2]
```diff
@@ -1,3 +1,4 @@
+os: unstable
```
Comment (TravisEz13) [inline appveyor.yml]: Can you add a comment here that this causes the use of WMF5?
[appveyor yml docs](http://www.appveyor.com/docs/appveyor-yml)

Comment (arkasha) [inline appveyor.yml]: Done.

**Pull request merged**